medical_appointment/data/medical_appointment_sequence.xml

```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015-TODAY
+	     LasLabs [https://laslabs.com]
```

Comment:
It's best to make these explicit lines. Also, please drop my `-TODAY` - that's BS. We can make is 2015-2016 for mine, and 2016 for yours.

Doing this allows the copyrights to be maintained individually as time progresses, instead of our two copyrights being bound to each other. Minor nuances, but good practice 😉